src/components/Jobs/JobDescriptionTab.tsx

```diff
@@ -0,0 +1,87 @@
+import { Card, Divider, Group, Text } from '@mantine/core';
+
+import type { Job, WorkerMetadata } from '@/types/job';
+import { getUtcFromEpoch } from '@/utils/string';
+
+type props = {
+  jobData: { job: Job };
+};
+
+function JobDescriptionTab({ jobData }: props) {
+  const workers = jobData.job.workerMetadataList.map((workerMeta: WorkerMetadata) => (
+    <Card bg="gray.1" withBorder padding={0} radius="md" key={workerMeta.workerIndex}>
+      <Group position="apart" m={10}>
+        <Text>{workerMeta.jobId}</Text>
+        <Text>
+          CPU: <b> 2 </b> Ram: <b> 4000 MB </b> Disk: <b>100 MB</b> Network: <b>128 MBPS</b>
```

Comment:
Is this meant to be hardcoded for now? If so, leave a TODO note or something so it is not missed.